tests/Tests/200__monitor_and_manage/220__grafana/220__test_verify_port_forwarding_grafana.robot

```diff
@@ -0,0 +1,48 @@
+*** Settings ***
+Documentation   Verify that MT-SRE can connect to Grafana using port-forwarding
+...
+...             = Variables =
+...             | PROC             | Required |        Is used  to get the process id|
+
+Library        OperatingSystem
+Library        Process
+Library        SeleniumLibrary
+
+
+*** Variables ***
+${PROC}     none
+
+
+*** Test Cases ***
+Verify that MT-SRE can connect to Grafana using port-forwarding
+    [Documentation]  Verify Grafana webiste is using port-forwarding
+    [Tags]   Tier2
+     ...     ODS-754
+
+    # Port Forwarding
+    ${PROC}    Start Process    oc -n redhat-ods-monitoring port-forward $(oc get pods -n redhat-ods-monitoring | grep grafana | awk '{print $1}' | head -n 1) 3001  shell=True  # robocop: disable
+    # It takes aroung 7 sec for the website to come up
+    Sleep  7s
+    ${response} =   Run   curl -I "http://localhost:3001"   # robocop: disable
+    Run Keyword If    "HTTP/1.1 200 OK" not in $response    Fail
+    ...       ELSE     Log To Console    SRE can connect to Grafana using prot-forwarding"
+
+    Open Browser  http://localhost:3001  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+    Wait Until Element Is Visible     //div[@class="sidemenu-item dropdown"]
+    Run Keywords
+    ...    Click Element    //div[@class="sidemenu-item dropdown"]
+    ...    AND
+    ...    Click Element    //*[text()="Search"]
+
+    Wait Until Element Is Visible    //*[contains(@aria-label,"Jupyterhub SLIs")]
+
```

Comment:
Line 31~37 could be a keyword which can be used by people in future who is working with Grafana even though they do not have to have port forwarding since it is common for grafana  ODS dashboard. Login will differ for normal (route url) and port forwarding(local host) but the dashboard will be same IMO

Reply:
In the future we'll create a new GrafanaLibrary with keywords for this. I think right now it's ok to have it in the test

@mwaykole  You can consider to access to http://localhost:3001/api/health and get the grafana version